### PR TITLE
Features

### PR DESCRIPTION
**Pull request by qwibbler**
# Capstone project for code review.

- [ ] Personalize the content of your page.
- [ ] Follow the design guidelines.
- [ ] Build the home page.
- [ ] Build the about page.
- [ ] Mobile: up to 768px wide.
- [ ] Desktop: 768px or wider.
- [ ] The home page has a link in the menu to the about page.
- [ ] The logo in the header links to the home page.
- [ ] When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- [ ] The mobile menu has a close (X) button that closes the menu.
- [ ] The section "Featured speakers" created dynamically in JavaScript.
- [ ] Use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.
- [ ] Give appropriate credit to the author. 

[Loom Video](https://www.loom.com/share/ee565db93f1347c0a63a2a4c6b47f3fc)

## Update:

- [ ] Separate About page from home page.
- [ ] Add Program.
- [ ] Add Logo page.
- [ ] Add Past Events page.

[Updated Loom Video](https://www.loom.com/share/51d70e362e3b452d8143d0e0d47f7c6e)

### CODE REVIEW

**Comment by qwibbler** [comment]
Day 1, finished!

- [ ] Home Page (html, css) all finished except background incomplete.
- [ ] About Page (html, css) all finished.
- [ ] Nav Bar (html, css) all finished
- [ ] Nav Bar hamburger (js) open close all finished except testing close on link and add close on touch outside.
- [ ] Readme initialized.

**Comment by qwibbler** [comment]
End Day 2

- [ ] Add background pictures.
- [ ] Add scrollspy.
- [ ] Add features page, html, css, js.
- [ ] Change stylings accordingly.

Todo:
- [x] Add footer.
- [x] Add page by page screenshot.
- [ ] Record video.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        for their rights and take a step forward for freedom.</p>
+      <h3>2021.11.09 (TUE)</h3>
+      <h4>@ Allama Iqbal University, Lahore</h4>
+    </section>
+
+    <section class="about" id="aboutpage">
+      <h5>"Welcome, Poets All!"</h5>
+      <h2>ALLAMA IQBAL DAY<br>POETRY FESTIVAL 2021</h2>
+      <hr class="desktop">
+      <p>
```
Comment (rotimiazeez) [inline index.html]: - [x] Please note that the project contains the `Home` page and the `About` page👍 and they are separated with the possibility to navigate between the pages, there's a link from the `Home` page to the `About` page and the opposite too👌😉.  Also, both pages contain the same navbar and footer😎✔.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            <li><a id="homelink" href="#homepage">Iqbal Day</a></li>
+            <li class="mobile empty">_empty_</li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </header>
+
+  <main>
+    <section class="home" id="homepage">
```
Comment (rotimiazeez) [inline index.html:59]: - [x] Kindly note that your home page should contain a middle section with a column that includes a section header, a divider, five individual articles, each with an image, title, and description, and a line of text at the bottom👀, same as the given [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)😊.
- Something like this👇:
screenshot (16)

Comment (rotimiazeez) [reply index.html:59]: Kindly know that on mobile the middle section article list is organized in a vertical column😋 and on desktop, the article list is organized in a horizontal row.
- for example
screenshot (17) screenshot (16)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+      <h1>ALLAMA IQBAL DAY<br>POETRY FESTIVAL<br>2021</h1>
+      <p>A joyful day celeberating a great man with the dream of Pakistan
+        and an independent state for the Muslims of the subcontinent.
+        His golden words had the power to encourage the people to fight
+        for their rights and take a step forward for freedom.</p>
+      <h3>2021.11.09 (TUE)</h3>
+      <h4>@ Allama Iqbal University, Lahore</h4>
+    </section>
+
+    <section class="about" id="aboutpage">
```
Comment (rotimiazeez) [inline index.html]: - [x] Please make sure the about page contains a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo👌 and a bottom section with a centred column that includes a section header, a divider, a line of text, and two images with text centred over them😉.
- design example
screenshot (19)

Comment (rotimiazeez) [reply index.html]: Kindly note that on mobile the bottom section images are organized in a vertical column✔ and on desktop, the bottom section images are organized in a horizontal row🙄. 
- example
screenshot (20) screenshot (21)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @qwibbler 👋, 

### Good job so far!👏👏


### SUMMARY❤
- Good README file👏
- No linter error ✔
- Good PR description👍
- Friendly UI✌😎

### There are some issues that you still need to work on to go to the next project but you are almost there!💯

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Since it wasn't asked by the previous review I will mark this as optional but it is highly recommended since it was part of the given design. Please add this part of the footer as well:
Screenshot from 2021-11-18 19-45-44


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**